AdminUi/apps/admin_ui/lib/core/widgets/app_title.dart

```diff
@@ -21,7 +21,7 @@ class AppTitle extends StatelessWidget {
           TextSpan(
             children: [
               TextSpan(text: 'enmeshed', style: textStyle.copyWith(fontWeight: FontWeight.bold)),
-              const TextSpan(text: ' Backbone Admin UI', style: textStyle),
+              const TextSpan(text: ' Backbone AdminUI', style: textStyle),
```

Comment:
missing translation

Reply:
For this text is the same, Timo's explanation was that this was a fixed texts and it should not be translated.

Reply:
If you (@jkoenig134 and @Siolto) think differently here, feel free to say so. I was just under the impression that "Backbone Admin UI" will never be translated.

Reply:
I'm with you!

Reply:
On the other hand, it might be useful to extract the "product name" into some globally accessible thing, so that it's the same everywhere, and we can easily change it. I don't know though if you should do this with a constant, or with a translation.

Reply:
If it is planned to product name changes whenever language changes, then I recommend using localization. On the other hand it can be easily reverted to use constant.